quic/api/QuicTransportBase.cpp

```diff
@@ -1636,10 +1636,19 @@ QuicTransportBase::notifyPendingWriteOnStream(StreamId id, WriteCallback* wcb) {
   if (!qStream->writable()) {
     return folly::makeUnexpected(LocalErrorCode::STREAM_CLOSED);
   }
+
+  if (wcb == nullptr) {
+    return folly::makeUnexpected(LocalErrorCode::INVALID_WRITE_CALLBACK);
+  }
   // Add the callback to the pending write callbacks so that if we are closed
   // while we are scheduled in the loop, the close will error out the callbacks.
-  if (!pendingWriteCallbacks_.emplace(id, wcb).second) {
-    return folly::makeUnexpected(LocalErrorCode::INVALID_WRITE_CALLBACK);
+  auto existing_wcb_pair = pendingWriteCallbacks_.emplace(id, wcb);
+  if (existing_wcb_pair.second) {
```

Comment:
if second is true, that already means the new callback isn't the same as previous callback.

Reply:
oops, I meant `!second`. Will fix shortly